README.md

```diff
@@ -2,36 +2,50 @@
-
+| Site | Owner | Description | Data Type |
+|:----|:---|:---|:----|
+| [cohealthdata.dphe.state.co.us](http://cohealthdata.dphe.state.co.us) | Colorado Health and Environmental Data | | |
+| [coepht.dphe.state.co.us](https://www.cohealthdata.dphe.state.co.us/) | Colorado Environmental Public Health Tracking| | |
+| [cogcc.state.co.us](http://cogcc.state.co.us) | Oil and Gas Conservation Commission; also see [airwatergas.org](http://airwatergas.org) | | |
+| [colorado.gov/apps/oit/transparency/index.html](https://www.colorado.gov/apps/oit/transparency/index.html) | Colorado Transparency Online Project | | |
+| [colorado.gov/pacific/revenue/statistics](http://colorado.gov/pacific/revenue/statistics) | Department of Revenue | | |
+| [coloradohealthinstitute.org/data](http://coloradohealthinstitute.org/data) | Colorado Health Institute | | |
+| [data.colorado.gov](http://data.colorado.gov) | Colorado Information Marketplace / State of Colorado | | |
+| [data.opencolorado.org](http://data.opencolorado.org) | OpenColorado data catalog | | |
+| [datausa.io/profile/geo/colorado](http://datausa.io/profile/geo/colorado) | Data USA | | |
+| [data-cdphe.opendata.arcgis.com](http://data-cdphe.opendata.arcgis.com) | Colorado Department of Public Health and Environment | | |
+| [denvermetrodata.org](http://denvermetrodata.org) | Shift Research Lab | | |
+| [demography.dola.colorado.gov/data](http://demography.dola.colorado.gov/data) | Department of Local Affairs / Demography Office | | |
+| [developer.nrel.gov](http://developer.nrel.gov) | National Renewable Energy Laboratory | | |
+| [dtdapps.coloradodot.info/otis](http://dtdapps.coloradodot.info/otis) | Colorado Department of Transportation / Online Transportation Info System | | |
+| [drcog.org/services-and-resources/data-maps-and-modeling](http://drcog.org/services-and-resources/data-maps-and-modeling) | Denver Regional Council of Governments | | |
+| [hub.arcgis.com/datasets?q=denver](http://hub.arcgis.com/datasets?q=denver) | ArcGIS Hub | | |
+| [lexisnexis.com/hottopics/colorado](http://lexisnexis.com/hottopics/colorado) | Colorado Constitution, Revised Statues, Court Rules | | |
+| [geodata.co.gov](https://geodata.co.gov/) | Colorado GeoData Cache | | |
+| [rtd-denver.com/Developer.shtml](http://rtd-denver.com/Developer.shtml) | Regional Transportation District | | |
+| [sos.state.co.us](http://sos.state.co.us) | Secretary of State | | |
+| [water.state.co.us/DataMaps/DataSearch/Pages/DataSearch.aspx](http://water.state.co.us/DataMaps/DataSearch/Pages/DataSearch.aspx) | Department of Natural Resources | | |
+
+## County Organizations With Open Data
+| Site | Owner/County | Description | Data Type |
+|:----|:---|:---|:----|
+| | | | |
+
+## City Organizations With Open Data
+| Site | Owner/City | Description | Data Type |
+|:----|:---|:---|:----|
+| | | | |
```

Comment:
I just made my own fork after @chooliu merged yours, and added my lines. I think that's the proper way to do it (relatively new to collaborating on Github) ?